var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -23,20 +23,40 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
+from glob import glob
 
 
 class Libxsmm(Package):
-    '''LIBXSMM is a library for small dense and small sparse matrix-matrix
-    multiplications targeting Intel Architecture (x86).'''
+    '''Library targeting Intel Architecture
+    for small, dense or sparse matrix multiplications,
+    and small convolutions.'''
 
     homepage = 'https://github.com/hfp/libxsmm'
-    url      = 'https://github.com/hfp/libxsmm/archive/1.4.3.tar.gz'
```

Comment:
Hmm, removing the URL and then `spack uninstall --all libxsmm` (after some previous installation) gives the following:

```
NoURLError: Package Libxsmm has no version with a URL.
```

So I guess you mean to leave the "old" URL will still do the right anyways. Anyhow, I will keep the change as reverting it from the rest of the patch is not worth the time.